chore: Allow to configure editor download urls (#2948)
codenvy/che-docs: modules/administration-guide/pages/configuring-editors-download-urls.adoc

:_content-type: PROCEDURE
:description: Configuring editors download urls
:keywords: administration guide, dashboard, editors
:navtitle: Configuring editors download urls

[id="configuring-editors-download-urls"]

> = Configuring editors download urls

Reviewer: ```suggestion
= Configuring editors download URLs
```

Reviewer: Reason for the above update: According to the IBM style guide's conventions for abbreviations, initialisms like URL (Uniform Resource Locator) should be in uppercase

> Learn how to configure editors download urls. This feature is valuable in air-gapped environments where editors cannot be retrieved from the public internet.

Reviewer 2: Same concern as in https://github.com/eclipse-che/che-operator/pull/2026/files#r2269472787 
we must clarify which exact editors could benifit from this property

> Learn how to configure editors download urls. This feature is valuable in air-gapped environments where editors cannot be retrieved from the public internet. Currently, this option is intended only for JetBrains editors and should not be used for other editor types.

Reviewer: ```suggestion
This procedure describes how to configure download URLs for editors. This feature is valuable in air-gapped environments where editors cannot be retrieved from the public internet. Currently, this option is intended only for JetBrains editors and should not be used for other editor types.
```

Reviewer: Reason for the above update: The IBM style guide recommends direct, user-focused language, especially for technical procedures.

.Prerequisites

> * An active `{orch-cli}` session with administrative permissions to the {orch-name} cluster. See {orch-cli-link}.

Reviewer: ```suggestion
* You have an active `_orch-cli_` session with administrative permissions to the `{orch-name}` cluster. For more information, see {orch-cli-link}.
```

> * `jq`. See link:https://stedolan.github.io/jq/download/[Downloading `jq`].

Reviewer: ```suggestion
* The `jq` command-line tool is installed. For more information, see link:https://stedolan.github.io/jq/download/[Downloading `jq`].
```
